ra/registration-authority.go

```diff
@@ -640,6 +640,21 @@ func (ra *RegistrationAuthorityImpl) checkCertificatesPerNameLimit(names []strin
 	return nil
 }
 
+func (ra *RegistrationAuthorityImpl) checkCertificatesPerNameSetLimit(names []string, limit cmd.RateLimitPolicy, regID int64) error {
+	count, err := ra.SA.CountValidNameSets(names)
+	if err != nil {
+		return err
+	}
+	core.LowerAndSort(names)
```

Comment:
This seems to be the only motivating reason for LowerAndSort being in core, and I'm not so sure it's enough reason. Maybe let's get rid of it?